pkg/infra/pulumi_aws/deploylib.ts

```diff
@@ -620,7 +620,7 @@ export class CloudCCLib {
         publishers: string[],
         subscribers: string[]
     ): aws.sns.Topic {
-        let topic = `${this.name}_${path.replace(/[^0-9a-zA-Z_-]/g, '-')}_${varName}_${event}`
+        let topic = `${this.name}_${varName}_${event}`
```

Comment:
Why remove the path? Is `varName` not the emitter's variable name anymore (ie, is it the ID)? Didn't this need to match with something in the runtime as well?

Reply:
so the path isnt in the runtime and i believe that varName is actually the ID and not truly the varName. Let me double check because if it is the varname we could have conflicts. I can also rename the variable to pubsubID so it makes more sense